### DISCUSSION TITLE
Cover Art Obscures Text

### DISCUSSION THREAD

**Discussion by john-** [jcorporation/myMPD#505] [Q&A] [answered]
myMPD version: 7.0.2

When album cover art has dark areas it obscures the playlist.   This makes it hard to read the text.

Example:
![mypd_screen2](https://user-images.githubusercontent.com/2359125/120673881-31b4d480-c459-11eb-9fc2-0ac9c0e614e3.png)

I don't know if this is specific to my install.   There is the same behavior in Firefox on Linux and Chrome on a Mac laptop.   

**To Reproduce**

1. Play an album with a cover that has large areas that are dark in color.

**Expected behavior**

The album cover art should not make it hard to read the text in the UI.

**Screenshots**

**Server plattform (please complete the following information):**
 - OS: Linux
 - OS version: TBD
 - Platform: Arm
 - MPD version: 0.22.3

**Client plattform (please complete the following information):**
 - Device: 2 laptops
 - OS: Fedora 34 and Mac OS
 - Browser Firefox and Chrome
 - Browser version: Various

**Comment by jcorporation** [comment, marked as answer]
In the default config there is a css filter activated for the background image to prevent this problem.

You can reset it to default in the settings. I personally prefer grayscale with 10% opacity.

https://developer.mozilla.org/en-US/docs/Web/CSS/filter

**Reply by john-** [reply]
Thanks.

I changed config and restarted myMPD:

![grayscale_config](https://user-images.githubusercontent.com/2359125/120710932-612d0680-c484-11eb-8618-63f9bbebb086.png)

However, maybe I have another problem because the text is barely visible:

![text_barely_visible](https://user-images.githubusercontent.com/2359125/120710978-72761300-c484-11eb-96cc-c84e035d9456.png)

**Reply by jcorporation** [reply]
Set the theme to dark if you set a black background color.

**Reply by john-** [reply]
Perfect...thanks!
